src/app/components/search/search.component.html

```diff
@@ -109,7 +116,14 @@ <h1 class="govuk-label-wrapper">
                 <label class="ds_label" for="range-date-to">To</label>
                 <p class="ds_hint-text">For example, 31/01/2023.</p>
                 <div class="ds_input__wrapper">
-                  <input formControlName="date_to" id="range-date-to" type="text" class="ds_input ds_input--fixed-10" />
+                  <input
+                    formControlName="date_to"
+                    id="range-date-to"
+                    name="range-date-to"
+                    type="text"
+                    class="ds_input ds_input--fixed-10"
+                    (change)="onChange($event, 'date_to')"
```

Comment:
It's not a deal-breaker but I'd rather see the value passed through to the method in the component. I feel like it makes the code cleaner and easier to understand.